src/MahApps.Metro/MahApps.Metro/Styles/Controls.PasswordBox.xaml

```diff
@@ -369,7 +369,7 @@
                                 SnapsToDevicePixels="{TemplateBinding SnapsToDevicePixels}" />
                         <Grid x:Name="PART_InnerGrid" Margin="2">
                             <Grid.ColumnDefinitions>
-                                <ColumnDefinition Width="*" />
+                                <ColumnDefinition Width="Auto" />
```

Comment:
This is wrong value if buttons alignment is right.

Reply:
Add a new style?